src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -110,9 +110,16 @@ private SealightsCLIBuildStep resolveFromCLiRunner() {
 
     private CommandMode.ConfigView getConfigParams(){
         CommandMode.ConfigView configView = (CommandMode.ConfigView) commandMode;
-        configView.setAppName(cliRunner.getAppName());
-        configView.setBranchName(cliRunner.getBranchName());
-        configView.setBuildName(cliRunner.getBuildName());
+        if(!StringUtils.isNullOrEmpty(cliRunner.getAppName())) {
+            configView.setAppName(cliRunner.getAppName());
+        }
+        if(!StringUtils.isNullOrEmpty(cliRunner.getBranchName())) {
+            configView.setBranchName(cliRunner.getBranchName());
+        }
+        // Cli runner might initiate with default build name
+        if(cliRunner.getBuildName() != null && !(cliRunner.getBuildName() instanceof CommandBuildName.DefaultBuildName)) {
```

Comment:
Is there a reason not to use the default build name?
Which name is it?

Reply:
Sorry i mismatched between stuff, ignore it.
Default build name is job build number